daffodil-io/src/main/scala/org/apache/daffodil/processors/charset/CharsetCompilerRegistry.scala

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Suggest renaming this class from `CharsetCompilerRegistry.scala` to `BitsCharsetDefinitionRegistry.scala`.